test/unit/specs/components/SelectedLicense.spec.js

```diff
@@ -0,0 +1,28 @@
+import { shallowMount } from '@vue/test-utils'
+import SelectedLicense from '@/components/SelectedLicense.vue'
+
+describe('SelectedLicense.spec.js', () => {
+    let cmp
+
+    beforeEach(() => {
+        cmp = shallowMount(SelectedLicense, {
+        // Be aware that props is overridden using `propsData`
+            propsData: {
+                fullLicenseName: 'Attribution',
```

Comment:
Please format fullLicenseName as "{name} 4.0 International". For example, BY would be "Attribution 4.0 International", BY-ND would be "Attribution-NoDerivatives 4.0 International". This is how the prop is actually formatted when passed to the component.